server/routes/applications.js

```diff
@@ -87,6 +87,33 @@ router.get('/manage-applications/:appId/view/:env', (req, res, next) => {
   });
 });
 
+router.get('/manage-applications/:appId/view/:env/:keyName', (req, res, next) => {
```

Comment:
I thought Env was being added to the end of the URL?

Reply:
That version is still there, this is for the specific case of having just been redirected from the update key route and needing to display the key name in the confirmation message

Reply:
Is that too much of an impediment as far as you're concerned or are you ok if I go ahead and merge?

Reply:
I think we should be consistent. And its not much to change.

But I guess the sky won't fall if you merge?
Why aren't we doing this via a confirm thing the same way we did delete?

Reply:
It's a different thing. It's not confirm that you want to update, it's re-display the app overview page with a message confirming that the key was updated

Reply:
Oh hold on, are you saying you'll be ok if I do exactly the same thing but just change the orders of the request params?

Reply:
Sorry I was being silly, I thought you were objecting to the presence of this new route

Reply:
changed